tsdb/mapper.go

```diff
@@ -306,7 +338,7 @@ func (lm *LocalMapper) nextChunkRaw() (*MapperOutput, error) {
 // for the current tagset. Tagsets are always processed in the same order as that
 // returned by AvailTagsSets(). When there is no more data for any tagset nil
 // is returned.
-func (lm *LocalMapper) nextChunkAgg() (*MapperOutput, error) {
+func (lm *LocalMapper) nextChunkAgg() (interface{}, error) {
```

Comment:
Yeah, I think this change to `MapperOutput` was made by @corylanou and it looked fine at the time as it seemed a better use of the type system. It seems you had trouble with it, and are reverting it back. OK.